v3/src/components/inspector-panel.scss

```
@@ -7,7 +7,7 @@ $inspector-panel-width: 50px;
   top: 0;
   width: $inspector-panel-width;
   min-height: 50px;
-  z-index: 10;  // Will need to be higher to accommodate Leaflet
+  z-index: 50;
```

Out of curiosity, what was the symptom that led you to bump up the `z-index` here? I'd like to consider whether there might be a better solution than `z-index` inflation.

---

This was mainly for the window expand/contract icon to not appear on top of the palette.

---

There may be a better solution lurking here, but for now make it `15` and we can put off a deeper analysis for later.
```suggestion
  z-index: 15;
```